Gemfile

```diff
@@ -6,6 +6,7 @@ gemspec
 # Specify the same dependency sources as the application Gemfile
 gem("activesupport", "~> 5.2")
 gem("railties", "~> 5.2")
+gem("vernier", "~> 0.3.1") if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("3.2.1")
```

Comment:
kind of ugly, but lets us support both ruby versions prior to vernier support, and post.